web/src/test/java/org/cbioportal/web/util/StudyViewFilterApplierTest.java

```diff
@@ -63,6 +63,8 @@
     private SampleListService sampleListService;
     @Mock
     private GenericAssayService genericAssayService;
+    @Mock
+    private TreatmentService treatmentService;
```

Comment:
Is this necessary? Will you create more test cases for `StudyViewFilterApplierTest`?